entries/r/restorecord.com.json

```
@@ -0,0 +1,13 @@
+{
+    "RestoreCord": {
+        "domain": "restorecord.com",
+        "url": "https://restorecord.com/login/",
+        "tfa": [
+            "totp"
+        ],
+        "categories": [
+            "backup",
+			"communication"
```

```suggestion
            "backup"
```

I would classify it as just `backup`